db/column_family.h

```diff
@@ -232,8 +236,23 @@ struct SuperVersion {
   // that needs to be deleted in to_delete vector. Unrefing those
   // objects needs to be done in the mutex
   void Cleanup();
-  void Init(ColumnFamilyData* new_cfd, MemTable* new_mem,
-            MemTableListVersion* new_imm, Version* new_current);
+  void Init(
+      ColumnFamilyData* new_cfd, MemTable* new_mem,
+      MemTableListVersion* new_imm, Version* new_current,
+      std::shared_ptr<const SeqnoToTimeMapping>&& new_seqno_to_time_mapping);
+
+  // Make a shared copy of the seqno to time mapping object referred to in this
+  // SuperVersion. To be used by the new SuperVersion to be installed after this
+  // one if seqno to time mapping does not change in between these two
+  // SuperVersions.
+  std::shared_ptr<const SeqnoToTimeMapping> CopySeqnoToTimeMapping() {
```

Comment:
I find the name "Copy..." slightly confusing because it's copying the pointer not the object, while above the "shared copy of the DB's seqno to time mapping" is an object copy.

Reply:
Thank you for noticing this. I have changed the name to "ShareSeqnoToTimeMapping" to indicate this just shares the ownership of existing copy.